python/GangaLHCb/Lib/GaudiRunApp/GaudiRunAppHandlers.py

```diff
@@ -0,0 +1,345 @@
+import os
+import shutil
+from datetime import datetime
+import tempfile
+import tarfile
+import random
+import threading
+import uuid
+
+from Ganga.Core import ApplicationConfigurationError
+from Ganga.GPIDev.Adapters.ApplicationRuntimeHandlers import allHandlers
+from Ganga.GPIDev.Adapters.IRuntimeHandler import IRuntimeHandler
+from Ganga.GPIDev.Adapters.StandardJobConfig import StandardJobConfig
+from Ganga.GPIDev.Base.Proxy import getName
+from Ganga.GPIDev.Lib.File.File import File, ShareDir
+from Ganga.GPIDev.Lib.File.FileBuffer import FileBuffer
+from Ganga.GPIDev.Lib.File.LocalFile import LocalFile
+from Ganga.GPIDev.Lib.File.OutputFileManager import getWNCodeForOutputPostprocessing
+from Ganga.Utility.Config import getConfig
+from Ganga.Utility.files import unique
```

Comment:
I think this should be `from Ganga.Utility.util import unique`? I'm surprised this didn't show up in testing though.